foreman/requirements.in

```diff
@@ -12,7 +12,7 @@ djangorestframework>=3.11.2
 # Can't go above 6.X without upgrading elasticsearch
 django-elasticsearch-dsl==6.5.0
 django-elasticsearch-dsl-drf>=0.17.1
-elasticsearch-dsl>=6.1.0
+elasticsearch-dsl>=6.1.0,<7.0.0
```

Comment:
It looks like you need to run `pip-compile` here too:

https://github.com/AlexsLemonade/refinebio/blob/8780d82400d88df280401aa6efa767652378987d/foreman/requirements.txt#L18,L19